src/widget/wtrackmenu.cpp

```diff
@@ -914,6 +917,17 @@ class ImportMetadataFromFileTagsTrackPointerOperation : public mixxx::TrackPoint
 
 } // anonymous namespace
 
+void WTrackMenu::slotUpdateReplaygain() {
+    if (!m_pTrack) {
+        qDebug() << "track pointer nullptr, returning";
+        return;
+    }
+    if (m_deckGroup.isEmpty()) {
+        qDebug() << "deck group not set";
```

Comment:
return.

Reply:
done

Reply:
Same here. Log spam that until we switch from qDebug() to qInfo().